### ISSUE TITLE
Detect OS and advertise mobile app

### ISSUE THREAD

**Issue by moT01** [freeCodeCamp/freeCodeCamp#43350]
@azatecas This is a good point. @nhcarrigan perhaps we could notify them "Hey it looks like your using Android. We recommend installing the freeCodeCamp Android app for a better coding experience. Or at least reading this: https://www.freecodecamp.org/news/freecodecamp-mobile/"

_Originally posted by @QuincyLarson in https://github.com/freeCodeCamp/freeCodeCamp/issues/41578#issuecomment-822058867_

**Comment by listentothefrog** [comment]
Hi @moT01, thanks for pointing out this issue. I will take a stab at this issue and open pr :)

**Comment by Tanmay1407** [comment]
Hi! I do have a good hold over backend technologies but being new to Open Source can anyone guide me from where do I start?
As I would like to contribute to the following issue.
Thanks :-)

**Comment by Sembauke** [comment]
Hey @Tanmay1407,

You can read our [contributor's guide ](https://contribute.freecodecamp.org/#/)to get started with contributing to the fcc repository.

Happy coding 😄

**Comment by pawan43563** [comment]
Based on os we have to give pop up for installation right?

**Comment by ieahleen** [comment]
@pawan43563 yes, and to the article about coding keyboards, in case people can't or don't want the app

**Comment by pawan43563** [comment]
okay would love to contribute can you assign me?

**Comment by ieahleen** [comment]
@pawan43563 
An issue with the _help wanted_ or _first timers only_ label is open for contribution. The first comprehensive PR created will be reviewed and merged. We typically do not assign issues to anyone other than long-time contributors.

If you would like to contribute, and have not read the contributors docs, please do so here: https://contribute.freecodecamp.org/#/

If you have any issues with contributing, be sure to join us on the [contributors channel](https://chat.freecodecamp.org/channel/contributors), or on the [contributors sub-forum](https://forum.freecodecamp.org/c/contributors/3)

**Comment by EhEhRon91** [comment]
Think I have a fix for this, but not 100% sure if everything is done exactly how it should. Do I make a pull request and then have it reviewed (and make changes) or is it possible to have the branch reviewed beforehand? Fairly new to open source and would appreciate any advice!

**Comment by ieahleen** [comment]
@EhEhRon91 open a PR so that it can be reviewed, if changes are needed or there is something wrong etc, the review is for that
